components/Grid.js

```diff
@@ -7,7 +7,7 @@ import propTypes from '@styled-system/prop-types';
 import styled from 'styled-components';
 import { color, compose, flexbox, grid, layout, space, typography } from 'styled-system';
 
-const boxProps = compose(space, color, layout, typography, flexbox);
+const boxProps = compose(space, color, layout, typography, flexbox, grid);
```

Comment:
Why not using the `Grid` component directly?

Reply:
Here, I'm mapping the grid children props so I can use `grid-column` later in a `Box` component.
```jsx
<Grid>
  <Box gridColumn={1}></Box>
</Grid>
```